radsat-sk/operation/message/RMessage.c

```diff
@@ -5,6 +5,7 @@
  */

```

Comment:
append your name

Reply:
I didn't add anything. Ty said it just needed a PR then I will be adding the error manager functionality to the rest of the code in another PR.

Reply:
I will add my name if you really think I need to, but I mean to only add my name if I add code.